app/models/ranker/base.rb

```diff
@@ -0,0 +1,10 @@
+module Ranker
+  class Base
+    def self.apply_stats(*args)
```

Comment:
Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used. You can also write as `apply_stats(*)` if you want the method to accept any arguments but don't care about them.